packages/debugger/src/tokens.ts

```diff
@@ -484,6 +486,26 @@ export namespace IDebugger {
     }
   }
 
+  /**
+   * Debugger sidebar interface.
+   */
+  export interface ISidebar extends Widget {
+    /**
+     * Add panel at the end of the sidebar.
+     */
+    addPanel(widget: Widget): void;
```

Comment:
Thinking about the names of these two methods a bit more, since they will be part of the API.

At first it looks like there could be a chance of confusing the "panel" part of `addPanel` with a Lumino Panel.

However "Panel" is definitely valid in this case for the sidebar. But I wonder if we should consider alternatives such as `addSection` and `addItem`.

Reply:
I agree, "Panel" is appropriate; however, like you mentioned because it may get confused with Lumino `Panel` maybe we should change it. I like `Section` it sounds much better than `Item` or `Panel`.

I looked at VSCode documentation and it seems there the "Panels" are referred to as "sections" as well. For example "VARIABLES section of the Run view" (see https://code.visualstudio.com/docs/editor/debugging ).

Reply:
ok, so we could then go for "section"?

Just checked in the JupyterLab code base, and the word section is used in different places (at least internally) to designate similar areas in the UI. For example in the notebook tools and the running panel.

Reply:
In PyCharm, they call it "pane" or "tab":

- https://www.jetbrains.com/help/pycharm/part-1-debugging-python-code.html#watching
- https://www.jetbrains.com/help/pycharm/debug-tool-window-variables.html

But their UX is slightly different.

Reply:
In my opinion `Section` is a good and descriptive word. `Pane` is a good as well, but it may be too close to `Panel` :).  I have mixed feeling about `Tab` especially with the layout that we currently have.
I don't have a strong opinion on this, so we can go either way if you have a strong preference?

Reply:
I think the terms of art that we use in other places, `addItem(...)` and `insertItem(...)` are appropriate here. In the toolbar, `addItem()` and `inserItem()` accept `Widget` isntances so I think it's analogous to here.

In the toolbar, there is this note, which I think we should mirror here:
```
The item can be removed from the toolbar by setting its parent to `null`.
```

After that change, I think this is a solid API. Thank you for patiently iterating on this @mnowacki-b! 🚀